### PR TITLE
docs(linux): AM62AX/J7: Update href link and meta-edgeai tag for 10.1

### PR DESCRIPTION
**Pull request by sinha-shreyash**

### CODE REVIEW

File: source/devices/AM62AX/index_Edge_AI.rst
Comment (StaticRocket) [inline source/devices/AM62AX/index_Edge_AI.rst:1]: Is this file really necessary? This seems to be the only device with a page like this and it contains the same information in the device specific release notes.

**Review by StaticRocket — CHANGES_REQUESTED** [review]

**Review by cshilwant — COMMENTED** [review]
For so meta-edgeai, Are we going 10.01.00.05 tag for AM62A and 10.01.00.04 for J7 family?

**Comment by sinha-shreyash** [comment]
> So for meta-edgeai, Are we going 10.01.00.05 tag for AM62A and 10.01.00.04 for J7 family?
Yes

File: source/devices/AM62AX/index_Edge_AI.rst
Comment (sinha-shreyash) [reply source/devices/AM62AX/index_Edge_AI.rst:1]: Yeah, this device is specific for edgeai use case, other devices (J7 Family) are specific for ADAS use cases.

Release notes, points to the meta-edgeai layer and patches for kernel and u-boot, not to the user guide

**Review by cshilwant — DISMISSED** [review]

File: source/linux/index_Edge_AI.rst [hunk 1]
```diff
@@ -12,14 +10,18 @@ Edge AI
 
    For more information on edge AI software stack, refer `Edge AI Documentation <https://software-dl.ti.com/processor-sdk-linux/esd/AM62AX/10_01_00/exports/edgeai-docs/common/sdk_overview.html#>`_
 
-.. ifconfig:: CONFIG_part_variant in ('J721E')
+.. ifconfig:: CONFIG_part_variant in ('TDA4VM')
```
Comment (aniket-l) [inline source/linux/index_Edge_AI.rst:13]: @sinha-shreyash 
This is not right. There is not CONFIG_part_variant TDA4VM
TDA4VM uses 'J721E' as CONFIG_part_variant and 'edgeai' as CONFIG_image_type

Same comment below in the rest of the file

**Review by aniket-l — CHANGES_REQUESTED** [review]

File: source/linux/index_Edge_AI.rst (hunk 1 above)
Comment (sinha-shreyash) [reply source/linux/index_Edge_AI.rst]: updated

**Review by StaticRocket — DISMISSED** [review]

**Review by StaticRocket — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]
